Admin.js

```diff
@@ -11,7 +11,7 @@
 
```

Comment:
- eslint: Line 224 exceeds the maximum line length of 80.
- eslint: Unexpected var, use let or const instead.
- eslint: All 'var' declarations must be at the top of the function scope.
- eslint: Identifier 'filters_container' is not in camel case.
- eslint: Unexpected string concatenation.
- eslint: Strings must use doublequote.
- eslint: Strings must use doublequote.